### PR TITLE
{bio} [goolf-1.4.10] [goolf-1.7.20] Pysam-0.9-Python-2.7.5, Pysam-0.9-Python-2.7.11 (REVIEW)

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

File: easybuild/easyconfigs/p/Pysam/Pysam-0.9.0-goolf-1.4.10-Python-2.7.5.eb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+toolchainopts = {'optarch': False, 'pic': False}
+
+parallel = 1
+
+python = 'Python'
+pyver = '2.7.5'
+pyshortver = '.'.join(pyver.split('.')[:2])
+versionsuffix = '-%s-%s' % (python, pyver)
```
Comment (boegel) [inline easybuild/easyconfigs/p/Pysam/Pysam-0.9.0-goolf-1.4.10-Python-2.7.5.eb]: @pescobar please use `%(pyver)s`?

File: easybuild/easyconfigs/p/Pysam/Pysam-0.9.0-goolf-1.4.10-Python-2.7.5.eb [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    }),
+]
+
+modextrapaths = {
+    'PYTHONPATH': ['lib/python%s/site-packages' % pyshortver],
+}
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%s/site-packages/pysam-%s-py%s-linux-x86_64.egg' % (pyshortver, version, pyshortver)]
```
Comment (boegel) [inline easybuild/easyconfigs/p/Pysam/Pysam-0.9.0-goolf-1.4.10-Python-2.7.5.eb]: @pescobar please use `%(pyshortver)s` and `%(version)s`:

```python
dirs': ['lib/python%(pyshortver)s/site-packages/pysam-%(version)s-py%(pyshortver)s-linux-x86_64.egg'],
```

**Comment by boegel** [comment]
@pescobar conflict on `zlib` version:

```
Conflict found for dependencies of Pysam-0.9.0-goolf-1.4.10-Python-2.7.5: zlib-1.2.7-goolf-1.4.10 vs zlib-1.2.8-goolf-1.4.10
```

**Comment by boegel** [comment]
@pescobar ping?

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login18.cluster.bc2.ch - Linux centos 6.5, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.6.6
See https://gist.github.com/87c27b76fb627611cdcb64d04135dc1b for a full test report.

**Comment by pescobar** [comment]
Test report by @pescobar
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
login18.cluster.bc2.ch - Linux centos 6.5, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.6.6
See https://gist.github.com/d7c78f36d43f7e9bba96694c113b1cc6 for a full test report.

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
login18.cluster.bc2.ch - Linux centos 6.5, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.6.6
See https://gist.github.com/89e7d37578939f3b0210140d3eedf5cf for a full test report.

**Comment by boegel** [comment]
(close/open to trigger Travis)

**Comment by boegel** [comment]
@pescobar for some reason, Travis isn't picking up your last commit... Can you merge `develop` in here to fix that, maybe?

**Comment by boegel** [comment]
@pescobar:

```
Conflict found for dependencies of Pysam-0.9.0-goolf-1.4.10-Python-2.7.5:
zlib-1.2.7-goolf-1.4.10 vs zlib-1.2.8-goolf-1.4.10
```

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 1 out of 1 (3 easyconfigs in this PR)
login18.cluster.bc2.ch - Linux centos 6.5, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.6.6
See https://gist.github.com/ff5d09660057482615a924237c7b662c for a full test report.

File: easybuild/easyconfigs/p/Pysam/Pysam-0.9.0-goolf-1.7.20-Python-2.7.11.eb [hunk 3]
```diff
@@ -29,7 +29,7 @@ exts_defaultclass = 'PythonPackage'
 exts_filter = ("python -c 'import %(ext_name)s'", '')
 
 exts_list = [
-    ('Cython', '0.23.4', {
+    ('Cython', '0.24', {
```
Comment (boegel) [inline easybuild/easyconfigs/p/Pysam/Pysam-0.9.0-goolf-1.7.20-Python-2.7.11.eb]: @pescobar why change this?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 2 out of 3 (3 easyconfigs in this PR)
node2160.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/81be7fea58ac98bfe8c764aa0d5bcaaa for a full test report.

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
login18.cluster.bc2.ch - Linux centos 6.5, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.6.6
See https://gist.github.com/9ec53c314fee9e22f59a1345343ccba0 for a full test report.

**Comment by boegel** [comment]
hmm, dafuq...

```
gcc -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -O2 -march=native -I/user/scratch/gent/vsc400/vsc40023/easybuild_REGTEST/SL6/sandybridge/software/OpenBLAS/0.2.6-gompi-1.4.10-LAPACK-3.4.2/include -I/user/scratch/gent/vsc400/vsc40023/easybuild_REGTEST/SL6/sandybridge/software/ScaLAPACK/2.0.2-gompi-1.4.10-OpenBLAS-0.2.6-LAPACK-3.4.2/include -I/user/scratch/gent/vsc400/vsc40023/easybuild_REGTEST/SL6/sandybridge/software/FFTW/3.3.3-gompi-1.4.10/include -fPIC -Ipysam -I. -Ihtslib -I/user/scratch/gent/vsc400/vsc40023/easybuild_REGTEST/SL6/sandybridge/software/Python/2.7.5-goolf-1.4.10/include/python2.7 -c pysam/chtslib.c -o build/temp.linux-x86_64-2.7/pysam/chtslib.o -Wno-unused -Wno-strict-prototypes -Wno-sign-compare -Wno-error=declaration-after-statement
gcc: error: pysam/chtslib.c: No such file or directory
gcc: fatal error: no input files
compilation terminated.
```

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 1 year.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**